themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md

```diff
@@ -213,7 +344,33 @@ We also need to create `Container` resources for the frontend and mongo
 containers. Put the `mongo_container` declaration just above the `backend_container` one, and the
```

Comment:
There's some differences here with snake_case and CamelCase for declarations for Python and TypeScript. I think it may confuse some readers. Messy, I know. Maybe that inline chooser might help here?

Reply:
Yep, another oversight, good spot. I'll add a `typescript` block for it.